package.json

```
@@ -29,6 +29,15 @@
     "onView:openhands.quickActions"
   ],
   "contributes": {
+    "configuration": {
```

Biome (2.1.2)

[error] The key configuration was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)